modules/swagger-codegen/src/main/resources/php/Configuration.mustache

```diff
@@ -123,51 +109,6 @@ class Configuration
     protected $tempFolderPath;
 
     /**
-     * Indicates if SSL verification should be enabled or disabled.
```

Comment:
Settled then. I'll update this PR soon with default version (with guzzle as its easier to use than httplug) and add `--library=psr7` in near future.